### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by adityadeshpande09**
fixes: #271  
Added license information in the README file.

### CODE REVIEW

**Comment by adityadeshpande09** [comment]
@jacobsfletch  Can you please review and merge this PR ASAP.
Thanks !! 😄

**Comment by adityadeshpande09** [comment]
@jacobsfletch  Can you please review and merge this PR.

File: README.md [hunk 1]
```diff
@@ -45,4 +45,8 @@ The locally running app must run on `local.payloadcms.com:3000` because of http-
 127.0.0.1 local.payloadcms.com
 ```
 
-> On Mac you can find the hosts file at `/etc/hosts`. On Windows, it's at `C:\Windows\System32\drivers\etc\hosts`:
\ No newline at end of file
+> On Mac you can find the hosts file at `/etc/hosts`. On Windows, it's at `C:\Windows\System32\drivers\etc\hosts`:
+
+### License
+
+The Payload CMS website is available as open source under the terms of the [MIT license](https://github.com/payloadcms/website/blob/main/LICENSE).
```
Comment (jacobsfletch) [inline README.md:52]: @adityadeshpande09 can you remove "CMS" from this?
```suggestion
The Payload website is available as open source under the terms of the [MIT license](https://github.com/payloadcms/website/blob/main/LICENSE).
```

**Comment by adityadeshpande09** [comment]
Done !! 😄

**Comment by jacobsfletch** [comment]
Thanks again @adityadeshpande09 👍

**Pull request merged**